requirements.txt

```diff
@@ -1,3 +1,3 @@
-invoke
-pyyaml
+invoke==2.2.0
+pyyaml==5.4.1
```

Comment:
I did not put the last version just to test dependabot will work

Reply:
Ahah that's a good idea!